docs/source/configuration.rst

```diff
@@ -68,8 +68,8 @@ use_nav_sat
 
 use_landmarks
   If enabled, subscribes to `cartographer_ros_msgs/LandmarkList`_ on the topic
-  "landmarks".  Landmarks must be provided in this case, and the information
-  will be included in SLAM.
+  "landmarks".  Landmarks must be provided as `cartographer_ros_msgs/LandmarkEntry`_ within `cartographer_ros_msgs/LandmarkList`_.  If `cartographer_ros_msgs/LandmarkEntry`_ data is provided the information
+  will be included in the SLAM accoding to the id of the `cartographer_ros_msgs/LandmarkEntry`_. The `cartographer_ros_msgs/LandmarkList`_ should be provided at a sample rate comparable to the other sensors.  The list can be empty but has to be provided as cartographer strictly time orders sensor data in order to make the landmarks deterministic. For debugging reasons it is possible to set the trajectory builder option "collate_landmarks" to false and allow for an non-deterministic but also non-blocking approach.
```

Comment:
I thought so myself but the id in the cartographer_ros_msgs is written lower-case. So I would leave it lower-case. But we could also change it to upper-case

Reply:
I have no strong opinion on this, so let's leave it and wait for another opinion

Reply:
"provided because Cartographer" -> Good idea
"a non-deterministic" -> Thank you

Reply:
How can I make these changes in the GitHub Webinterface? I am seriously asking since I never really had to use the thing?

Reply:
I decided to write the ID upper-case since I believe that ID should always be written upper-case to prevent confusion. It might be advisable to change the cartographer_ros_msg but I have in no way enough expertise to suggest that.